app/src/main/java/woowacourse/movie/notification/PendingIntentGenerator.kt

```diff
@@ -0,0 +1,33 @@
+package woowacourse.movie.notification
+
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+
+object PendingIntentGenerator {
+
+    private var requestCode = 0
+
+    private fun getFlag() = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+        PendingIntent.FLAG_MUTABLE
+    } else {
+        PendingIntent.FLAG_UPDATE_CURRENT
+    }
```

Comment:
```suggestion
    private val flag 
        get() = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
                PendingIntent.FLAG_MUTABLE
            } else {
                PendingIntent.FLAG_UPDATE_CURRENT
            }
```
property 에서 제공하는 getter 를 사용하시면 좀 더 자연스러울 것 같아요~!
getter / setter 의 경우 함수를 만들기보다 property 를 활용해보는 연습을 해보셔도 좋겠습니다 ㅎㅎ